packages/top-app-bar/README.md

```diff
@@ -60,7 +54,7 @@ If you decide to just pass an element your markup should resemble this:
 
 ### Action Items
 
-Similar to the [navigation icon](#navigation-icon), it can be `<a>`, `<i>`, `<svg>`, `<image>`, `<span>`, etc. If you decide to pass a non-native `React.Component`, you need to append the prop `className` to the className prop on the parent element. As an example please see the [MaterialIcon Component](../material-icon/index.js).
+Similar to the [navigation icon](#navigation-icon), it can be `<a>`, `<i>`, `<svg>`, `<image>`, `<span>`, etc.
 
 ```js
 import MyIcon from './myIcon';
```

Comment:
Instead of having this code snippet which shows how to add className, just link to the docs/guidelines.md